spec/controllers/v1.0/graphql_controller_spec.rb

```diff
@@ -45,25 +52,35 @@
       before { approver_access }
 
       it 'selects attributes in workflows' do
-        post "#{api_version}/graphql", :headers => headers, :params => graphql_source_query
+        post "#{api_version}/graphql", :headers => headers, :params => graphql_simple_query
         expect(response.status).to eq(403)
       end
     end
+  end
+
+  describe 'a graphql query with id params' do
+    let(:user) { instance_double(UserContext, :access => access, :rbac_enabled? => true, :params => params, :graphql_params => graphql_params) }
+    let(:graphql_params) { double("graphql_params", :id => "#{parent_request.id}", :filter => nil) }
 
     context 'requests with admin role' do
       let(:headers) { headers_with_admin }
       before { admin_access }
 
       it 'return requests' do
-        post "#{api_version}/graphql", :headers => headers, :params => graphql_requests_query
+        post "#{api_version}/graphql", :headers => headers, :params => graphql_id_query
```

Comment:
@hsong-rh Do you have a case where where one ID is visible but another ID is not accessible?

Reply:
Just added a case for regular user. Now one is visible, the other is not